packages/dashboard-core-plugins/.gitignore

```diff
@@ -1,34 +1,12 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
-# dependencies
-/node_modules
+# misc
+
+
+
 
-# testing
-/coverage
 
-# production
-/build
-/dist
 
-# misc
-.vscode
-.DS_Store
-.env.local
-.env.development.local
-.env.test.local
-.env.production.local
-.project
-.settings/
-.eslintcache
-.stylelintcache
-
-/public/vs
-
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-
-src/**/*.css
 
 cypress/screenshots
 cypress/videos
```

Comment:
You can remove these too and delete this file. We don't use cypress in testing any more